### DISCUSSION TITLE
part-ii/2018-47/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#42] [Question comments]
Part II 2018 Differential Geometry: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2018-47/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

part (c). no. $S\cup\{p\}$ is not necessarily a smooth surface

</details>

**Comment by eha1234** [comment]
<details><summary>Spoiler </summary>

example for part (c) try attaching a cone to a circle (with some perturbation to make it smooth). 
</details>
